.travis.yml

```diff
@@ -24,30 +35,67 @@ matrix:
   include:
     - os: linux
       env: MB_PYTHON_VERSION=2.7
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=2.7
+        - UNICODE_WIDTH=16
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.5
+        - NP_BUILD_DEP=numpy==1.9.3
+        - NP_TEST_DEP=numpy==1.9.3
+    - os: linux
+      env:
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP=numpy==1.11.3
+        - NP_TEST_DEP=numpy==1.11.3
     - os: osx
-      osx_image: xcode6.4
       language: generic
       env:
         - MB_PYTHON_VERSION=2.7
     - os: osx
-      osx_image: xcode6.4
       language: generic
       env:
         - MB_PYTHON_VERSION=3.5
+        - NP_BUILD_DEP=numpy==1.9.3
+        - NP_TEST_DEP=numpy==1.9.3
+    - os: osx
+      language: generic
+      env:
+        - MB_PYTHON_VERSION=3.6
+        - NP_BUILD_DEP=numpy==1.11.3
+        - NP_TEST_DEP=numpy==1.11.3
+
 before_install:
+    - if [ "$TRAVIS_BRANCH" == "master" ]; then
+          CONTAINER="pre-release";
```

Comment:
Is container env var used anywhere? Was it supposed to be used in wheelhouse uploader?

Reply:
Other repos use it in the wheelhouse_uploader command, to specify a different container for daily builds, than for release builds - e.g. https://github.com/MacPython/cython-wheels/blob/master/.travis.yml#L132

Reply:
Ok, thought so. I think I'll pr to change that? My thought it to build "dailies" on master and then release off off of tags or a branch with a specific regex : https://docs.travis-ci.com/user/customizing-the-build/#Safelisting-or-blocklisting-branches

Reply:
Right - I think that's the best model - build dailies off master, and release from branches.